lib/xcpretty/formatters/formatter.rb

```diff
@@ -16,7 +16,7 @@ def format_clean(project, target, configuration);         EMPTY; end
     def format_clean_target(target, project, configuration);  EMPTY; end
     def format_clean_remove;                                  EMPTY; end
     def format_compile(file_name, file_path);                 EMPTY; end
-    def format_compile_command(compiler_command);             EMPTY; end
+    def format_compile_command(compiler_command, file_path);  EMPTY; end
```

Comment:
Unused method argument - `compiler_command`. If it's necessary, use `_` or `_compiler_command` as an argument name to indicate that it won't be used. You can also write as `format_compile_command(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `file_path`. If it's necessary, use `_` or `_file_path` as an argument name to indicate that it won't be used. You can also write as `format_compile_command(*)` if you want the method to accept any arguments but don't care about them.<br>Use empty lines between defs.